kobotoolbox/kpi
# Add newsletter subscription boolean

## Checklist

1. [X] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [X] Ensure the tests pass
4. [X] Make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [X] Write a title and, if necessary, a description of your work suitable for publishing in our [release notes](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes

## Description

Newsletter subscription field for TOS feature

## Notes

Added `newsletter_subscription` boolean to `extra_details` which reflects whether the user is subscribed to the newsletter. 

To test:
- navigate to the create an account page (should see a new field with checkbox to subscribe to the newsletter)
- create a new user who is subscribed to the newsletter
- after confirming the email and signing in, you should see `newsletter_subscription: True` in the `/me` endpoint
